perl/lib/Utils/DbSnp2FormatInfo.pm

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/perl
+use 5.10.0;
+use strict;
+use warnings;
+use DDP;
+
+# Take a DbSNP 2 VCF file, and for each row, split the INFO field's FREQ data into separate INFO fields for each population
+package Utils::DbSnp2FormatInfo;
+
+our $VERSION = '0.001';
+
+use File::Basename qw/basename/;
+
+use Mouse 2;
+use namespace::autoclean;
+use Path::Tiny qw/path/;
+
+use Seq::Tracks::Build::LocalFilesPaths;
+
+# Exports: _localFilesDir, _decodedConfig, compress, _wantedTrack, _setConfig, logPath, use_absolute_path
+extends 'Utils::Base';
+
+sub BUILD {
+  my $self = shift;
+
+  my $localFilesHandler = Seq::Tracks::Build::LocalFilesPaths->new();
+
+  my $localFilesAref = $localFilesHandler->makeAbsolutePaths(
+    $self->_decodedConfig->{files_dir},
+    $self->_wantedTrack->{name},
+    $self->_wantedTrack->{local_files}
+  );
+
+  $self->{_localFiles} = $localFilesAref;
+}
+
+# TODO: error check opening of file handles, write tests
+sub go {
```

Comment:
This function seems like it's doing a lot of different things at a few different levels of abstraction.  One way to address that might be to break it up into helpers roughly along the lines of:

- input validation
- setting up filepaths
- the core nested loop where we process $in_fh into $output_data_fh
- updating the VCF header
- writing everything out and cleaning up

But I defer to your judgment as to whether the juice is worth the squeeze there?